### PR TITLE
Dev Branch 

### PR DESCRIPTION
**Pull request by LuisSalas94**
- Added index.html
- Added img/
- Added style.css
- Fixed linters errors
- Modified style.css
- Added image to README

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+.navbar .fa-bars {
+  color: var(--primary);
+  font-size: 16px;
+}
+
+.main {
+  background: url(img/header-shapes-mobile.svg);
+  background-repeat: no-repeat;
+  background-size: cover;
```
Comment (micheaol) [inline style.css:44]: Please you must stick to the design as much as possible (e.g. font, colors, images, margins) using the templates in [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

### Figma design:
figma

 ### Your design:
your-design

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @LuisSalas94 🥇 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 


## Required Changes ♻️
- _Check the comments under the review._
- Please add an image(project image) in the README.md file to make your README more professional.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by LuisSalas94** [comment]
Hi Micheaol 
Thanks for the feedback. I have already made the changes in the CSS and also added the image to the readme file.

image

Could tell me where can I ask for a re-review? 
Thanks

**Comment by micheaol** [comment]
@LuisSalas94,

Thank you for making the required changes.
Kindly request for re-review from your dashboard.

**Review by Cathella — APPROVED** [review]
## 🟢 Project Approved 💯 🎉

### Hi @LuisSalas94 👋🏻, 
 You did a tremendous job implementing the required changes 💪🏻.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional Changes

- [ ] [OPTIONAL] I think your `<header>` could use a little bit of more padding. If you try to keenly compare your design and the figma design 👇🏻. Can you see the difference 🤔?

| Your design | Figma design |
| --- | --- |
| image | image |

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**